README.md

```diff
@@ -70,6 +70,28 @@ See `example.py` for several usage examples.
 
 Documentation for The Blue Alliance's API can be found [here](https://www.thebluealliance.com/apidocs).
 
+## Write Functions
+Writing to The Blue Alliance requires an additional set of tokens that can be obtained [here](https://www.thebluealliance.com/request/apiwrite). 
+
+Since each set of tokens is unique to an event you will have the instantiate the TBA class like so:
+```
+tba = tbapy.TBA('key', 'ID', 'secret', 'event_key')
+```
+or update an existing TBA object like so:
+```
+tba.update_trusted('ID', 'secret', 'event_key')
```

Comment:
`update_trusted` seems like kind of opaque naming to me (just commenting, not a problem with this PR per se).

Reply:
My function naming habits are pretty bad, so I highly suggest you change it.

Reply:
I'm better at naming things than I am at actually programming, so I'll see what I can do. :)